### DISCUSSION TITLE
Convert PSD file with ZipWithPrediction(32 bit) compression to JPG format

### DISCUSSION THREAD

**Discussion by lpmnobre** [dlemstra/Magick.NET#674] [Help]
Hi!
I need to convert PSD files with ZipWithPrediction(32 bit) compression type to the JPG format. I have installed the zlib.net package in my Visual Studio Project, but I don't know to configure Magick.Net to use it as a delegate program. Is it possible to do that?
Thanks so much for your help!

**Comment by dlemstra** [comment]
The ZIP compression is handled within the library itself and doe not require the zlib.net package. The Magick.NET (and ImageMagick) library does not support reading PSD files that have ZipWithPrediction. An issue to add support for this has been created here: https://github.com/ImageMagick/ImageMagick/issues/455. It would be helpful if you could add a file that has ZipWithPrediction to that issues.

**Comment by dlemstra** [comment]
I was able to figure it out yesterday and the next version of Magick.NET will include support for 32 bit ZipWithPrediction files.

**Comment by lpmnobre** [comment]
Thank you very much for your prompt reply!
I will check with my client for a test case.
Congratulations on your excellent hard work!
It is amazing!

**Comment by dlemstra** [comment]
The new release has been published. Feel free to reach out if you run into any issues.

**Comment by lpmnobre** [comment]
I'm very impressed with your excellent work and dedication to this Project!
It worked very well! Congratulations!
Thanks so much!

Em dom., 21 de jun. de 2020 às 17:09, Dirk Lemstra <notifications@github.com>
escreveu:
